app/src/Components/Event/EventDetails.tsx

```diff
@@ -18,7 +18,7 @@ import EventTime from './EventTime'
 import ImageFullWidth from '../ImageFullWidth'
 import ImageSwiper from '../ImageSwiper'
 import Title from '../Title'
-import Video from '../Video'
+import VideoComponent from '../../NativeBase/Components/Video'
```

Comment:
No worries, I named this Video Component due to React Native Video having a Video component but have renamed that now instead.

Reply:
So probably the easiest way you could do it is in `Video.tsx` change the React Native Video import line to `import ReactNativeVideo from 'react-native-video'` Then use it as `<ReactNativeVideo>` in that file.

Then in `Video.tsx` you can export your component as `Video` and use it as `Video` everywhere else rather than needing to call it `VideoComponent`.  Hope that helps!